pkg/rpcmiddleware/discover/discover.go

```diff
@@ -49,15 +50,37 @@ func defaultPort(target string, port int) (string, error) {
 	return target, nil
 }
 
-var defaultPorts = map[bool]int{
+// DefaultPorts is a map of the default gRPC ports, with/without TLS.
+var DefaultPorts = map[bool]int{
 	false: 1884,
 	true:  8884,
 }
 
+// DefaultHTTPPorts is a map of the default HTTP ports, with/without TLS.
+var DefaultHTTPPorts = map[bool]int{
+	false: 1885,
+	true:  8885,
+}
```

Comment:
Not completely sure about these defaults. On local deployments we indeed use 1885/8885, but for online deployments all our guides use 80/443.

Reply:
Right. This should indeed be 80/443.